airflow/www/static/js/main.js

```diff
@@ -226,10 +255,14 @@ $(document).ready(() => {
   $.fn.datetimepicker.defaults.sideBySide = true;
   $('.datetimepicker').datetimepicker();
 
+  $('.filters .select2-chosen').each((idx, el) => { filterOpSelected(el); });
```

Comment:
```suggestion
  $('.filters .select2-chosen').each((idx, e) => { filterOpSelected(e); });
```
Looks like `el` is already defined in the file and causing linting issues.